src/NzbDrone.Update.Test/ProgramFixture.cs

```diff
@@ -32,8 +32,9 @@ public void should_throw_if_first_arg_isnt_an_int(string arg1, string arg2)
         }
 
         [Test]
-        public void should_call_update_with_corret_path()
+        public void should_call_update_with_correct_path_windows()
```

Comment:
We should update the test, the test wasn't updated when the code was updated to support additional parameters and allows Windows to only pass a single argument for backwards compatibility.

`const string ProcessPath = @"C:\NzbDrone\nzbdrone.exe";` should be `var processPath = @"C:\NzbDrone\nzbdrone.exe".AsOsAgnostic();` and pass in `processPath` as the 3rd argument (executing assembly).